### ISSUE TITLE
DeviantArt Favorites Gallery 

### ISSUE THREAD

**Issue by ghost** [RipMeApp/ripme#826]
* Ripme version:
* Java version: 1.8
* Operating system: Windows 10

* Exact URL you were trying to rip when the problem occurred:

https://www.deviantart.com/kerbal69/favourites/

* Please include any additional information about how to reproduce the problem:

## Expected Behavior

Expected to download the gallery found on the website DeviantArt

## Actual Behavior

RipMe is appears to be unable to process the url (I am assuming related to the domain changes done to deviantart in the recent history). Exact error message is: 

" Expected URL format: http://www,deviantart.com/username[/gallery/#####], got: https://www.deviantart.com/kerbal69/favourites/gallery "

(Sorry if this is a duplicate post, I could not find any with similar problems to this.) 
If you need anymore information please let me know.

**Comment by RevStrangeHope** [comment]
Is there any update on this issue? I been trying to find a way to backup my favorites

**Comment by cyian-1756** [comment]
> Is there any update on this issue? 

Nope. Unfortunately DeviantArt is a total pain to work with so no one has managed to get the ripper working well

**Comment by rautamiekka** [comment]
> > Is there any update on this issue?
> 
> Nope. Unfortunately DeviantArt is a total pain to work with so no one has managed to get the ripper working well

Please explain.

**Comment by rautamiekka** [comment]
> @rautamiekka
> 
> They have insane rate limiting on requests for the api
> 
> Even if your using the api you still have to do some html scraping to get a photos UUID (Which makes the entire api pointless)
> 
> It's a pain and a half getting a full sized image
> 
> They make you log in to view some things which makes development a pain (No one likes handling cookies)
> 
> I could go on but [this](https://github.com/DeviantArt/DeviantArt-API/issues/153) explains it nicely

Gotta admit, that's bad, and not just pretty bad, it's really bad.

I had been coding the program part itself of a DA toolkit in Python which'd do many things but I can't find the code anywhere anymore, and although I had already decided to resort to HTML and JSON scraping it might be a whole lot of problems getting it to work, especially without the help of Seledium ...

**Comment by metaprime** [comment]
[Mega-Thread] DeviantArt ripper is broken; yes we know -- de-duping all other DeviantArt issues to this one. https://github.com/RipMeApp/ripme/issues/2063
